clients/cli/src/proto/nexus.orchestrator.rs

```diff
@@ -1,34 +1,4 @@
 // This file is @generated by prost-build.
-/// Register a User.
-#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
should we git-ignore these auto-gen files? Or at least have github ignore them?

Reply:
I think need to be checked in:
- so it's possible to build without protobuf compiler
- so we can see if the output changed without the proto file changes, and how

I generally skip reading them when reviewing.

The proto doesn't change very often for CLI but happy to revisit in the future if it gets annoying.

Reply:
yeah that's fine. I belive there's a way to get github to not expand them though. Forgot how though.

Reply:
I believe you're talking about .gitattributes and that's a totally reasonable idea. I've added it to the PR.